WiitarThing/App.xaml

```diff
@@ -27,12 +27,14 @@
         <SolidColorBrush x:Key="BrushAntemBlue" Color="#FF7CA3CA"/>
         <SolidColorBrush x:Key="BrushJohnTheRevelator" Color="#FF5A6E83"/>
         <BitmapImage x:Key="ProIcon" CreateOptions="IgnoreImageCache,IgnoreColorProfile" CacheOption="OnLoad" UriSource="Images/ProController_white_32.png"/>
-        <BitmapImage x:Key="CCIcon"  CreateOptions="IgnoreImageCache,IgnoreColorProfile" CacheOption="OnLoad" UriSource="Images/Classic_white_32.png"/>
+        <BitmapImage x:Key="CCIcon" CreateOptions="IgnoreImageCache,IgnoreColorProfile" CacheOption="OnLoad" UriSource= "Images/Classic_white_32.png"/>
         <BitmapImage x:Key="CCPIcon" CreateOptions="IgnoreImageCache,IgnoreColorProfile" CacheOption="OnLoad" UriSource="Images/ClassicPro_white_32.png"/>
-        <BitmapImage x:Key="WIcon"   CreateOptions="IgnoreImageCache,IgnoreColorProfile" CacheOption="OnLoad" UriSource="Images/wiimote_white_32.png"/>
-        <BitmapImage x:Key="WNIcon"  CreateOptions="IgnoreImageCache,IgnoreColorProfile" CacheOption="OnLoad" UriSource="Images/WiimoteNunchuk_white_32.png"/>
-        <BitmapImage x:Key="UIcon"   CreateOptions="IgnoreImageCache,IgnoreColorProfile" CacheOption="OnLoad" UriSource="Images/unknown.png"/>
-        <BitmapImage x:Key="GTRIcon" CreateOptions="IgnoreImageCache,IgnoreColorProfile" CacheOption="OnLoad" UriSource="Images/GHWT_Wii_Guitar.png"/>
-        <BitmapImage x:Key="DRMIcon" CreateOptions="IgnoreImageCache,IgnoreColorProfile" CacheOption="OnLoad" UriSource="Images/GHWT_Wii_Drums.png" />
+        <BitmapImage x:Key="WIcon" CreateOptions="IgnoreImageCache,IgnoreColorProfile" CacheOption="OnLoad" UriSource=  "Images/wiimote_white_32.png"/>
+        <BitmapImage x:Key="WNIcon" CreateOptions="IgnoreImageCache,IgnoreColorProfile" CacheOption="OnLoad" UriSource= "Images/WiimoteNunchuk_white_32.png"/>
+        <BitmapImage x:Key="UIcon" CreateOptions="IgnoreImageCache,IgnoreColorProfile" CacheOption="OnLoad" UriSource= "Images/unknown.png"/>
+        <BitmapImage x:Key="WGTIcon" CreateOptions="IgnoreImageCache,IgnoreColorProfile" CacheOption="OnLoad" UriSource= "Images/GHWT_Wii_Guitar.png"/>
+        <BitmapImage x:Key="WDRIcon" CreateOptions="IgnoreImageCache,IgnoreColorProfile" CacheOption="OnLoad" UriSource="Images/GHWT_Wii_Drums.png" />
+        <!-- Turntable image by Sk8er5000, CC BY-SA 3.0: https://commons.wikimedia.org/wiki/File:DJ_Hero_Turntable_PS3.jpg -->
+        <BitmapImage x:Key="WTBIcon" CreateOptions="IgnoreImageCache,IgnoreColorProfile" CacheOption="OnLoad" UriSource= "Images/DJHero_Turntable.png"/>
```

Comment:
Would like the icon name changed, having the Wii prefix on everything is redundant 'cause it's a Wii peripheral program lol
```suggestion
        <BitmapImage x:Key="TTBIcon" CreateOptions="IgnoreImageCache,IgnoreColorProfile" CacheOption="OnLoad" UriSource= "Images/DJHero_Turntable.png"/>
```